pkg/plugins/platform/router.go

```diff
@@ -76,8 +79,20 @@ func StartDdevRouter() error {
 	_, err = f.WriteString(doc.String())
 	util.CheckErr(err)
 
+	// Stop router so we can test port. Of course, it may not be running.
+	err = dockerutil.ComposeCmd([]string{dest}, "-p", RouterProjectName, "down")
+	if err != nil {
+		return fmt.Errorf("failed to stop ddev-router: %v", err)
+	}
+	err = CheckRouterPorts()
+	if err != nil {
+		return fmt.Errorf("Unable to listen on required ports, %v,\nTroubleshooting suggestions at https://github.com/drud/ddev/blob/master/docs/users/troubleshooting.md", err)
```

Comment:
Please link to an anchor just for troubleshooting ports in this. Right now, the entire troubleshooting doc is great, but as it grows we don't we want to avoid sending people to a sea of information, particularly if we have already identified their problem.

Reply:
The actual usage is like this in StartDdevRouter():
```
	err = CheckRouterPorts()
	if err != nil {
		return fmt.Errorf("Unable to listen on required ports, %v,\nTroubleshooting suggestions at https://github.com/drud/ddev/blob/master/docs/users/troubleshooting.md", err)
	}

```
I assume that's what you'd want.

Reply:
I think I may not have been clear enough. Sorry!

As you know, this PR introduces a new troubleshooting.md document. We're linking to it right here.

This works well for now, as the document is small, and finding information in it is very easy. However, this troubleshooting.md document is likely to grow over time. As such, it would be great to create anchors in it and link directly to those. That way if we add 3 more pages of information to the troubleshooting document over time, the user is still getting sent to an anchor that covers their specific issue.  In this case, we'd want to link directly to the "Webserver ports are already occupied by another webserver" section of that file.

Reply:
Added anchor in 4ccd4621